engineering/multimedia/README.md

```diff
@@ -0,0 +1,108 @@
+The position of the slider will be updated with the same interval as the position label.
+When the user drags the slider it will change the position of the stream with a seek.
+The final design including the slider should like like:
+
+`Position: HH:mm:ss.SSS [-----------------|---------------] Duration: HH:mm:ss.SSS`
+
+The story is successful when the user starts playing a clip and the slider is updated with the relative position.
+The story is successful when the drags the slider and a seek is performed updating the postion of the stream.
+
+### Create timeline with a snapshots preview
+As a user I would like to visually preview the contents of the stream once its loaded, in the form of 10 thumbnails displayed in a timeline. Thumbnails are spaciated in time at 1/10th of the duration of the clip. The thumnails generation should happen in background and the timeline should be updated asynchonously with the new thumbnails as they are generated.
+Here is an example of the desired feature.
+
+<img src="https://raw.githubusercontent.com/fluendo/recruitment/master/content/timelinepreview1.png" alt="enable-repl" width="300">
+<img src="https://raw.githubusercontent.com/fluendo/recruitment/master/content/timelinepreview2.png" alt="enable-repl" width="300">
+
+In the snapshot project you will find a sample code to extract thumbnails from a video file.
+The story is successful when the user loads a new video file and the thumnails timeline is progressively generated with 10 thumnails.
+
+It's time to start hacking!
+
+![hacker](https://raw.githubusercontent.com/fluendo/recruitment/master/content/THEGif.gif)
+
+## Video Player project
+
+The Video Player project has 3 components:
+  * videoplayer-qt: sample video player using the QT framework
+  * videoplayer-gtk3: sample video player using the Gtk+3 framework
+  * snapshot: sample application to get snapshots from a video file
+  
+The sources of the video player are taken from the GStreamer project examples and tutorials with the intention to provide a very basic starting point to start implementing new features for the test.
+
+### Setup
+On a machine with a Debian-like OS, install the following packages:
+
+```
+sudo apt-get install cmake libqt4-dev qtgstreamer-plugins libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libboost-dev libqtgstreamer-dev libgles2-mesa-dev gstreamer1.0-libav libgdk-pixbuf2.0-dev gtk-3-dev
+```
+
+### Configure
+The project uses a CMake build system. To configure it enter either videoplayer-qt or videoplayer-gtk3 depending on the toolkit of your choice and configure it with:
+```
+cmake CMakeLists.txt
+```
+
+### Build
+Once configured type:
+```
+make
+```
+
+### Run
+To run the app launch:
+```
+./videoplayer
+```
+
+The player starts without any video file openned. Click the open button and select a video file.
```

Comment:
open according to https://dictionary.cambridge.org/grammar/british-grammar/open-or-opened